pkgs/applications/misc/dupeguru/default.nix

```diff
@@ -0,0 +1,62 @@
+{stdenv, python, gettext, qt5, fetchFromGitHub}:
+
+stdenv.mkDerivation rec {
+  pname = "dupeguru";
+  version = "4.0.4";
+
+  src = fetchFromGitHub {
+    owner = "arsenetar";
+    repo = "dupeguru";
+    rev = "${version}";
+    sha256 = "0ma4f1c6vmpz8gi4sdy43x1ik7wh42wayvk1iq520d3i714kfcpy";
+    fetchSubmodules = true;
+  };
+
+  build_python = python.withPackages (ps: [ps.setuptools]);
+
+  nativeBuildInputs = [ 
+    gettext 
+    build_python
+    qt5.wrapQtAppsHook
+  ];
+
+  app_python = python.withPackages (ps: with ps; [
+      pyqt5
+      send2trash
+      sphinx
+      polib
+      hsaudiotag3k
+  ]);
+
+  buildInputs = [
+    app_python
+  ];
+
+  buildPhase = ''
+    runHook preBuild
+    make i18n
+    make modules
+    make qt/dg_rc.py
+    make pyc
+    runHook postBuild
+  '';
+
+  installPhase = ''
+    runHook preInstall
+    make PREFIX=$out install --assume-old=all --assume-old=pyc
+    chmod +x $out/share/dupeguru/run.py
+    runHook postInstall
+  '';
+
+  doCheck = false;
+
+  postFixup = ''
+    wrapQtApp $out/bin/dupeguru
+  '';
+
+  meta = with stdenv.lib; {
+    description = "dupeGuru is a cross-platform GUI tool to find duplicate files in a system";
+    homepage = https://github.com/arsenetar/dupeguru;
+    license = licenses.bsd3;
+  };
```

Comment:
Maintainers? Platforms?

Reply:
Is there any instruction on how to find any fitable?  Or any list of nixpkgs maintainers?

Reply:
`platforms = platforms.linux;` if you tested it on Linux, `unix` if you can also test it on MacOS.

Generally, when you add a new package, you volunteer to become its maintainer (see `maintainers` entry in https://nixos.org/nixpkgs/manual/#sec-standard-meta-attributes). Or you can try finding someone interested in maintaining this in the community.